src/Symfony/Component/Clock/MockClock.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Clock;
+
+/**
+ * A clock that returns always the same date, suitable for testing time-sensitive logic.
```

Comment:
just a native english suggestions
```suggestion
 * A clock that always returns the same date, suitable for testing time-sensitive logic.
```

Reply:
thanks Ben, fixed